### DISCUSSION TITLE
Open directory listings on https://cdn.jsdelivr.net/pyodide ?

### DISCUSSION THREAD

**Discussion by deeplook** [pyodide/pyodide#3563] [General]
Looking at the JupyterLite configuration I can tell that e.g. pandas is downloaded from https://cdn.jsdelivr.net/pyodide/v0.22.0/full/pandas-1.5.2-cp310-cp310-emscripten_3_1_27_wasm32.whl. But when trying to find out which other packages are available from that site by looking at https://cdn.jsdelivr.net/pyodide/v0.22.0/full/ I'm getting a "Forbidden" message. Would it be possible to "unhide" these items as this seems to be a solid source of information about which packages are available for JupyterLite.

**Comment by hoodmane** [comment]
It is annoying. @rth is the jsdelivr person here. But I think it may just be how jsdelivr works. There is an index in `repodata.json` which is likely suitable for your needs?

**Reply by rth** [reply]
Yes, JsDelivr doesn't allow us to serve HTML for security reasons. Indeed the way to go is to look at repodata.json (if you open it in Firefox it's nicely rendered) or `packages.json` for older Pyodide releases.

It's also possible to see the rendered list of packages in https://pyodide.org/en/stable/usage/packages-in-pyodide.html: there is a version selection widget on the bottom left.
